<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>depart</class>
 <widget class="QWidget" name="depart">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1214</width>
    <height>587</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>depart</string>
  </property>
  <widget class="QTabWidget" name="supp_8">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>1201</width>
     <height>561</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">alternate-background-color: qconicalgradient(cx:0, cy:0, angle:135, stop:0 rgba(255, 255, 0, 69), stop:0.375 rgba(255, 255, 0, 69), stop:0.423533 rgba(251, 255, 0, 145), stop:0.45 rgba(247, 255, 0, 208), stop:0.477581 rgba(255, 244, 71, 130), stop:0.518717 rgba(255, 218, 71, 130), stop:0.55 rgba(255, 255, 0, 255), stop:0.57754 rgba(255, 203, 0, 130), stop:0.625 rgba(255, 255, 0, 69), stop:1 rgba(255, 255, 0, 69));
background-color: qconicalgradient(cx:0, cy:0, angle:135, stop:0 rgba(255, 255, 0, 69), stop:0.375 rgba(255, 255, 0, 69), stop:0.423533 rgba(251, 255, 0, 145), stop:0.45 rgba(247, 255, 0, 208), stop:0.477581 rgba(255, 244, 71, 130), stop:0.518717 rgba(255, 218, 71, 130), stop:0.55 rgba(255, 255, 0, 255), stop:0.57754 rgba(255, 203, 0, 130), stop:0.625 rgba(255, 255, 0, 69), stop:1 rgba(255, 255, 0, 69));</string>
   </property>
   <property name="currentIndex">
    <number>5</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>departement</string>
    </attribute>
    <widget class="QGroupBox" name="groupBox">
     <property name="geometry">
      <rect>
       <x>19</x>
       <y>129</y>
       <width>271</width>
       <height>311</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 75 10pt &quot;MS Shell Dlg 2&quot;;</string>
     </property>
     <property name="title">
      <string>ajouter</string>
     </property>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>50</y>
        <width>56</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>id</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>110</y>
        <width>56</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>poste</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="ajouter_id_2">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>50</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="ajouter_poste_2">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>110</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="Ajouter_depart_2">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>210</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 0, 0);</string>
      </property>
      <property name="text">
       <string>ajouter</string>
      </property>
     </widget>
    </widget>
    <widget class="QTableView" name="table_depart">
     <property name="geometry">
      <rect>
       <x>415</x>
       <y>141</y>
       <width>691</width>
       <height>301</height>
      </rect>
     </property>
    </widget>
    <widget class="QRadioButton" name="tri_asc">
     <property name="geometry">
      <rect>
       <x>500</x>
       <y>90</y>
       <width>97</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 75 10pt &quot;MS Shell Dlg 2&quot;;</string>
     </property>
     <property name="text">
      <string>croissante</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="tri_desc">
     <property name="geometry">
      <rect>
       <x>780</x>
       <y>90</y>
       <width>141</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 75 10pt &quot;MS Shell Dlg 2&quot;;</string>
     </property>
     <property name="text">
      <string>decroissante</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonPDF_1">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>470</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 75 10pt &quot;MS Shell Dlg 2&quot;;
rgb background-color(0, 0, 255)</string>
     </property>
     <property name="text">
      <string>imprimer</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>modifier</string>
    </attribute>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>120</y>
       <width>271</width>
       <height>311</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 75 10pt &quot;MS Shell Dlg 2&quot;;</string>
     </property>
     <property name="title">
      <string>modifier</string>
     </property>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>50</y>
        <width>56</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>id</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>110</y>
        <width>56</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>poste</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="modifier_id">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>50</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="modifier_poste">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>110</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="modifier_depart">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>210</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">rgb background-color (0, 0, 255)</string>
      </property>
      <property name="text">
       <string>modifier</string>
      </property>
     </widget>
    </widget>
    <widget class="QPushButton" name="chercher_modif_depart">
     <property name="geometry">
      <rect>
       <x>450</x>
       <y>80</y>
       <width>311</width>
       <height>28</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 75 10pt &quot;MS Shell Dlg 2&quot;;
rgbbackground-color (0, 0, 255)</string>
     </property>
     <property name="text">
      <string>Chercher departement par identifiant</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="chercher_depart">
     <property name="geometry">
      <rect>
       <x>840</x>
       <y>80</y>
       <width>151</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);
font: 11pt &quot;MS Shell Dlg 2&quot;;</string>
     </property>
    </widget>
    <widget class="QTableView" name="table_Re_depart">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>220</y>
       <width>771</width>
       <height>91</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_4">
    <attribute name="title">
     <string>supprimer</string>
    </attribute>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>40</y>
       <width>461</width>
       <height>231</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 75 10pt &quot;MS Shell Dlg 2&quot;;</string>
     </property>
     <property name="title">
      <string>supprimer</string>
     </property>
     <widget class="QLineEdit" name="supprimer_depart">
      <property name="geometry">
       <rect>
        <x>560</x>
        <y>100</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QComboBox" name="comboBox_6">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>90</y>
       <width>191</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="supprimer_2">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>90</y>
       <width>171</width>
       <height>28</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 75 10pt &quot;MS Shell Dlg 2&quot;;</string>
     </property>
     <property name="text">
      <string>supprimer depart</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="sup_2">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>160</y>
       <width>113</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_6">
    <attribute name="title">
     <string>parking</string>
    </attribute>
    <widget class="QGroupBox" name="groupBox_4">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>70</y>
       <width>271</width>
       <height>311</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 75 10pt &quot;MS Shell Dlg 2&quot;;</string>
     </property>
     <property name="title">
      <string>ajouter</string>
     </property>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>50</y>
        <width>56</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>cin</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>110</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>immatricule</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="Ajouter_cin_2">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>50</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="ajouter_immatricule_2">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>110</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="Ajouter_parking_2">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>230</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">rgbbackground-color (0, 0, 255)</string>
      </property>
      <property name="text">
       <string>ajouter</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>160</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>num_tel</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="ajouter_num_tel_2">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>160</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QTableView" name="table_parking">
     <property name="geometry">
      <rect>
       <x>430</x>
       <y>90</y>
       <width>691</width>
       <height>301</height>
      </rect>
     </property>
    </widget>
    <widget class="QRadioButton" name="tri_asc_dpt">
     <property name="geometry">
      <rect>
       <x>470</x>
       <y>50</y>
       <width>131</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 75 10pt &quot;MS Shell Dlg 2&quot;;</string>
     </property>
     <property name="text">
      <string>croissante</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="tri_desc_dpt">
     <property name="geometry">
      <rect>
       <x>850</x>
       <y>50</y>
       <width>141</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 75 10pt &quot;MS Shell Dlg 2&quot;;</string>
     </property>
     <property name="text">
      <string>decroissante</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonPDF_2">
     <property name="geometry">
      <rect>
       <x>430</x>
       <y>420</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 75 10pt &quot;MS Shell Dlg 2&quot;;
rgbbackground-color (0, 0, 255)</string>
     </property>
     <property name="text">
      <string>imprimer</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_7">
    <attribute name="title">
     <string>modifier</string>
    </attribute>
    <widget class="QGroupBox" name="groupBox_5">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>60</y>
       <width>271</width>
       <height>311</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 75 10pt &quot;MS Shell Dlg 2&quot;;</string>
     </property>
     <property name="title">
      <string>modifier</string>
     </property>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>50</y>
        <width>56</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>cin</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>110</y>
        <width>91</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>immatricule</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="modifier_cin">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>50</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="modifier_immatricule">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>110</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="modifier_parking">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>230</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">rgbbackground-color (0, 0, 255)</string>
      </property>
      <property name="text">
       <string>modifier</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>160</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>num_tel</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="modifier_num_tel">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>160</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QPushButton" name="chercher_parking">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>70</y>
       <width>241</width>
       <height>28</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 75 10pt &quot;MS Shell Dlg 2&quot;;
rgbbackground-color (0, 0, 255)</string>
     </property>
     <property name="text">
      <string>chercher la voiture pae cin</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="recherche_parking_cin">
     <property name="geometry">
      <rect>
       <x>800</x>
       <y>70</y>
       <width>151</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
    </widget>
    <widget class="QTableView" name="table_Re_parking">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>200</y>
       <width>771</width>
       <height>91</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_8">
    <attribute name="title">
     <string>supprimer</string>
    </attribute>
    <widget class="QGroupBox" name="groupBox_6">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>210</y>
       <width>461</width>
       <height>271</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 75 10pt &quot;MS Shell Dlg 2&quot;;</string>
     </property>
     <property name="title">
      <string>supprimer</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_8">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>350</y>
       <width>191</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="supprimer_9">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>350</y>
       <width>151</width>
       <height>28</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 75 10pt &quot;MS Shell Dlg 2&quot;;</string>
     </property>
     <property name="text">
      <string>supprimer parking</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="sup_9">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>410</y>
       <width>113</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
